app/views/profile_field_groups/index.json.jbuilder

```diff
@@ -0,0 +1,21 @@
+json.profile_field_groups do
```

Comment:
I'm personally not a bif fan of APIs that directly return JSON arrays, so I decided to return an object with `profile_field_groups` as key.